### PR TITLE
New instructions have been added for use with fastif-cli

### PR DESCRIPTION
**Pull request by astesio**
#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,35 @@ How can you efficiently handle the resources of your server, knowing that you ar
 
 Enter Fastify. Fastify is a web framework highly focused on providing the best developer experience with the least overhead and a powerful plugin architecture. It is inspired by Hapi and Express and as far as we know, it is one of the fastest web frameworks in town.
 
+### Quick start
+
+Good tools make API's development quicker and easier to maintain that if you did everything by hand.
```
Comment (jsumners) [inline README.md]: => "Good tools make API development quicker and easier to maintain than doing everything manually.

File: README.md [hunk 2]
```diff
@@ -27,6 +27,35 @@ How can you efficiently handle the resources of your server, knowing that you ar
...
+
+The [Fastify CLI](https://github.com/fastify/fastify-cli) is a command line interface tool that can create a new project, add plugins, services and perform a variety of development tasks such as testing, and started the project.
```
Comment (jsumners) [inline README.md]: => "The [Fastify CLI] is a command line interface tool that can create new projects, manage plugins, and perform a variety of development tasks testing and running the application."

File: README.md [hunk 3]
```diff
@@ -27,6 +27,35 @@ How can you efficiently handle the resources of your server, knowing that you ar
...
+
+The goal in this guide is to build and run a simple Fastify project, using the [Fastify CLI](https://github.com/fastify/fastify-cli) while adhering to the Style Guide recommendations that benefit every Fastify project.
```
Comment (jsumners) [inline README.md]: Add a `,` after "[Fastify CLI]".

File: README.md [hunk 4]
```diff
@@ -27,6 +27,35 @@ How can you efficiently handle the resources of your server, knowing that you ar
...
+Open a terminal window.
+
+Generate a new project and default app by running the following command:
+
+
+```
+fastify generate
+```
+
+For more information, see the [Fastify CLI](https://github.com/fastify/fastify-cli) documentation.
```
Comment (jsumners) [inline README.md]: Extend the link to include the "documentation" word.

File: README.md [hunk 5]
```diff
@@ -27,6 +27,35 @@ How can you efficiently handle the resources of your server, knowing that you ar
...
+
+
+Otherwise do not want to use the [Fastify CLI](https://github.com/fastify/fastify-cli) you can do manualemnte too, as shown below
```
Comment (jsumners) [inline README.md]: Scratch this whole sentence.

File: README.md (hunk 1 above)
Comment (astesio) [reply README.md]: done.

File: README.md (hunk 2 above)
Comment (astesio) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (astesio) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (astesio) [reply README.md]: done

Comment (astesio) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (astesio) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (jsumners) [reply README.md]: You added the comma in the wrong sentence. It should be in the sentence associated with this comment.

File: README.md [hunk 6]
```diff
@@ -27,6 +27,31 @@ How can you efficiently handle the resources of your server, knowing that you ar
 
 Enter Fastify. Fastify is a web framework highly focused on providing the best developer experience with the least overhead and a powerful plugin architecture. It is inspired by Hapi and Express and as far as we know, it is one of the fastest web frameworks in town.
 
+### Quick start
+
+Good tools make API development quicker and easier to maintain than doing everything manually.
+
+The [Fastify CLI](https://github.com/fastify/fastify-cli), is a command line interface tool that can create new projects, manage plugins, and perform a variety of development tasks testing and running the application.
```
[the remark this answers is not in the document]
Comment (astesio) [reply README.md]: done.

File: README.md [hunk 7]
```diff
@@ -27,6 +27,31 @@ How can you efficiently handle the resources of your server, knowing that you ar
...
+
+The goal in this guide is to build and run a simple Fastify project, using the [Fastify CLI](https://github.com/fastify/fastify-cli) while adhering to the Style Guide recommendations that benefit every Fastify project.
```
[the remark this answers is not in the document]
Comment (astesio) [reply README.md]: done

File: README.md [hunk 8]
```diff
@@ -27,6 +27,31 @@ How can you efficiently handle the resources of your server, knowing that you ar
...
 
+### Quick start
+
+Good tools make API development quicker and easier to maintain than doing everything manually.
+
+The [Fastify CLI](https://github.com/fastify/fastify-cli) is a command line interface tool that can create new projects, manage plugins, and perform a variety of development tasks testing and running the application.
+
+The goal in this guide is to build and run a simple Fastify project, using the [Fastify CLI](https://github.com/fastify/fastify-cli), while adhering to the Style Guide recommendations that benefit every Fastify project.
+
+### Install
```
[the remark this answers is not in the document]
Comment (astesio) [reply README.md]: I made some corrections, I was finding it redundant, what do you think of this new way?

File: README.md [hunk 9]
```diff
@@ -73,6 +73,32 @@ fastify.listen(3000, (err, address) => {
...
 
+### Quick start with Fastify CLI
+
+Good tools make API development quicker and easier to maintain than doing everything manually.
+
+The [Fastify CLI](https://github.com/fastify/fastify-cli) is a command line interface tool that can create new projects, manage plugins, and perform a variety of development tasks testing and running the application.
+
+The goal in this guide is to build and run a simple Fastify project, using the [Fastify CLI](https://github.com/fastify/fastify-cli), while adhering to the Style Guide recommendations that benefit every Fastify project.
+
+The goal in this guide is to build and run a simple Fastify project, using the [Fastify CLI](https://github.com/fastify/fastify-cli), while adhering to the Style Guide recommendations that benefit every Fastify project.
```
Comment (delvedor) [inline README.md]: Why this is duplicated?

Comment (astesio) [reply README.md]: Oh man, so sorry, I sleeping yet. :(. Thanks

**Pull request merged**